wheel/src/run_generator.rs

```diff
@@ -4,6 +4,7 @@ use chia_protocol::to_json_dict::ToJsonDict;
 
 use chia::gen::conditions::{parse_spends, Spend, SpendBundleConditions};
 use chia::gen::validation_error::{ErrorCode, ValidationErr};
+use chia::gen::run_block_generator::run_block_generator as run_block_gen;
```

Comment:
I like the name `native_run_block_generator` or `rust_run_block_generator`. But this is not a blocker.

Reply:
yeah, that's a better name